package-lock.json

```diff
@@ -19187,6 +19187,7 @@
       "dependencies": {
         "@liveblocks/client": "1.3.0",
         "@liveblocks/core": "1.3.0",
+        "nanoid": "^4.0.2",
```

Comment:
Thinking about this… I think maybe we need to make this a `peerDependencies` instead, just to facilitate people that already may be using (a different version of) `nanoid` in their frontend projects?

Reply:
Do `peerDependencies` auto-install nowadays?

Reply:
They do these days!

<img width="783" alt="Screen Shot 2023-09-08 at 11 16 10@2x" src="https://github.com/liveblocks/liveblocks/assets/83844/bea2fdf6-2480-44dd-8f71-7bbffe57eff4">

Reply:
(And that makes a lot of sense.)